internal/cmd/run.go

```diff
@@ -47,17 +50,38 @@ func runCmd() *cobra.Command {
 					return err
 				}
 
-				for _, arg := range args {
-					block, err := lookupCodeBlock(blocks, arg)
-					if err != nil {
-						return err
+				if runAll || category != "" {
+					for _, block := range blocks {
+						if !block.ExcludeFromRunAll() {
```

Comment:
My feeling here is that using the --category applies the same behavior that --all so it will exclude it as well @sourishkrout do you have feedback on this one?

Reply:
That's fine, I just wanted to make sure that was an intentional choice.